sdk/keyvault/keyvault-keys/src/core/index.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference lib="es2015" />
 /// <reference lib="esnext.asynciterable"/>
 
-export { RestError } from "@azure/ms-rest-js";
+export { RestError } from "@azure/core-http";
```

Comment:
If you're editing the autogenerated code directly, is this coming from a new autogeneration or is it edits by-hand? If the former, great. If the latter, we should make sure we're generating this so we don't stomp on the change in a future update.